docs/src/components/landing-page/MainLayoutHeader.vue

```diff
@@ -1,119 +1,262 @@
 <template>
-  <q-header :class="dark? 'bg-white text-black-54':'bg-lp-dark text-white-54'" class="font-monserrat lp-header" ref="layoutHeader">
-    <q-toolbar
-      :class="{ 'shadow-bottom-small': $q.screen.xs, 'letter-spacing-25': $q.screen.lt.lg, 'letter-spacing-300': $q.screen.gt.md }"
-      class="primary-toolbar q-pl-lg q-pr-md justify-between items-stretch"
+  <transition
+    enter-active-class="animated slideInDown"
+    leave-active-class="animated fadeOut"
+  >
+    <q-header
+      v-if="primaryHeaderIsVisible"
+      ref="layoutHeader"
+      :class="dark? 'bg-white text-black-54':'bg-lp-dark text-white-54'"
+      class="font-monserrat lp-header"
     >
-      <q-btn v-if="$q.screen.xs" flat @click="$emit('update:modelValue', !modelValue)" round dense icon="menu" color="lp-primary"/>
-      <div v-if="$q.screen.gt.xs || !isSearchFieldActive" class="row justify-center items-center cursor-pointer" @click="$router.push({name: 'home'})">
-        <img v-if="$q.screen.sm" :src="`https://cdn.quasar.dev/logo-v2/svg/logo${!dark? '-dark':''}.svg`" width="48" height="48" alt="Quasar Logo">
-        <img v-else :src="`https://cdn.quasar.dev/logo-v2/svg/logo-horizontal${!dark? '-dark':''}.svg`" width="236" :height="$q.screen.xs? '24':'48'" alt="Quasar Logo">
-        <q-separator v-if="$q.screen.gt.xs" :color="dark? 'black-12':'lp-primary'" vertical class="q-ml-lg" />
-      </div>
+      <q-toolbar
+        :class="{ 'shadow-bottom-small': $q.screen.xs, 'letter-spacing-25': $q.screen.lt.lg, 'letter-spacing-300': $q.screen.gt.md, 'items-stretch': $q.screen.gt.xs }"
+        class="primary-toolbar q-pl-lg q-pr-md justify-between text-size-16"
+      >
+        <q-btn
+          v-if="$q.screen.xs"
+          color="lp-primary"
+          dense
+          flat
+          icon="menu"
+          round
+          @click="$emit('update:modelValue', !modelValue)"/>
+        <a
```

Comment:
Can this work with a `router-link` instead?
In other words, is there a way to make this work with `$router.push({name: 'home'})` style but keeping the "open in new tab" behaviour when clicking on the middle mouse button?
I think `router-link` with `:to="{name: 'home'}"` may automatically manage this, but I'm not sure

Reply:
Yes, it works with ```router-link```. I agree that it's a preferable solution since the one with ```a``` tag cannot work with named routes.